.github/workflows/test-unit.yml

```diff
@@ -220,6 +220,7 @@ jobs:
       - name: Install JS dependencies (only for Slow)
         if: matrix.type == 'Chrome Slow'
         run: |
+          git config --global url."https://github.com".insteadOf "ssh://git@github.com"
```

Comment:
not needed once https://github.com/atk4/image/pull/43/files#diff-8aad5bc8efe55f15feea83ad1cb1febdd84a072ef653e323afa72d9810890429R32 is merged